File: python_lesson/city_functions.py
def city_country(city, country, population=""):
    """ A City Inside a Country """ 
    places = {} 
    if population:
        places[city] = country
        places['population'] = population
        for cities, countries in places.items(): 
            place = (cities + ", " + countries + " - " + "population " +
            str(places['population'])) 
            return place
    else:
        places[city] = country 
        for cities, countries in places.items(): 
            place = cities + ", " + countries
            return place.title()

File: lua_lesson/city_functions.lua
function City_country(tbl)
    --[[ A City Inside a Country ]]--
    setmetatable(tbl, {__index={population=""}})
    local city, country, population = 
    tbl[1] or tbl.city, 
    tbl[2] or tbl.country, 
    tbl[3] or tbl.population
    Places = {}
    if population then
        Places[city] = country
        Places['population'] = population
        for cities, countries in pairs(Places) do
            Place = (cities..", "..countries.." - ".."population "..
            tostring(Places['population'])) 
            print(Place)
            return Place
        end
    else
        Places[city] = country 
        for cities, countries in pairs(Places) do
            Place = cities..", "..countries
            print(Place)
            return Place
        end
    end
end
